sonar-ws/src/main/protobuf/ws-qualityprofiles.proto

```diff
@@ -0,0 +1,46 @@
+// SonarQube, open source software quality management tool.
+// Copyright (C) 2008-2015 SonarSource
+// mailto:contact AT sonarsource DOT com
+//
+// SonarQube is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 3 of the License, or (at your option) any later version.
+//
+// SonarQube is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program; if not, write to the Free Software Foundation,
+// Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+syntax = "proto2";
+
+package sonarqube.ws.rules;
+
+option java_package = "org.sonarqube.ws";
+
+option java_outer_classname = "QualityProfiles";
+
+option optimize_for = SPEED;
+
+// WS api/qualityprofiles/search
+message WsSearchResponse {
+  repeated QualityProfile profiles = 1;
```

Comment:
I'll do that in another branch afterwards so stas will be able to do it when he will have time